th2_data_services/adapter.py

```diff
@@ -30,5 +30,6 @@ def change_pipeline_message(record: dict) -> Union[List[dict], dict]:
         new_record["body"] = body
         new_record["body"]["metadata"]["messageType"] = sub_msg_type
         new_record["messageType"] = sub_msg_type
+        new_record["messageId"] = f"{record.get('messageId')}.{index}"
```

Comment:
let's change to 
new_record["messageId"] = f"{record['messageId']}.{index}"

in order to get an exception instead of messageId = None.1